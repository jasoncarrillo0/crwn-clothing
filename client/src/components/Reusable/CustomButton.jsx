import React from 'react';
import { StyledCustomButton } from './StyledCustomButton.styles';

const CustomButton = ({children, ...otherProps}) => {
    return (
        <StyledCustomButton {...otherProps}>
            {children}
        </StyledCustomButton>
    )
}
export default CustomButton;