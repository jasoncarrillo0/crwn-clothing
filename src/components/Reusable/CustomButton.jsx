import React from 'react';
import './customButton.scss';

const CustomButton = ({children, inverted, isGoogleSignIn, ...otherProps}) => {
    return (
        <button 
            type="button" 
            className={
                `${isGoogleSignIn ? 'googleSignIn' : ''}
                ${inverted ? 'inverted' : ''} 
                customButton`
            } 
            {...otherProps}
        >
            {children}
        </button>
    )
}
export default CustomButton;