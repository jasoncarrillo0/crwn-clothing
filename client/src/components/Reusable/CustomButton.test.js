import { configure, shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import CustomButton from './CustomButton';

configure({adapter: new Adapter()});

it('expect to render CustomButton', () => {
    expect(shallow(<CustomButton>Test Button String</CustomButton>)).toMatchSnapshot();
})

