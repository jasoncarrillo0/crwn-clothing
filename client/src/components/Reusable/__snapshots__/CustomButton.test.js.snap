// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CustomButton 1`] = `ShallowWrapper {}`;
